src/main/java/com/scalar/db/storage/cassandra/CassandraTableMetadata.java

```diff
@@ -28,11 +37,57 @@ public CassandraTableMetadata(com.datastax.driver.core.TableMetadata tableMetada
             tableMetadata.getClusteringColumns().stream()
                 .map(ColumnMetadata::getName)
                 .collect(Collectors.toList()));
-    this.indexNames =
+    indexNames =
         ImmutableSet.copyOf(
             tableMetadata.getIndexes().stream()
                 .map(IndexMetadata::getTarget)
                 .collect(Collectors.toSet()));
+    columnDataTypes =
+        ImmutableMap.copyOf(
+            tableMetadata.getColumns().stream()
+                .collect(
+                    Collectors.toMap(
+                        ColumnMetadata::getName, c -> convertDataType(c.getType().getName()))));
+
+    Map<String, Scan.Ordering.Order> tmp = new HashMap<>();
```

Comment:
```suggestion
    Map<String, Scan.Ordering.Order> tmpClusteringOrder = new HashMap<>();
```

Reply:
It seems hard to avoid creating `tmp`, right?
I've thought about it but couldn't get a better solution.

Reply:
We can use `ImmutableMap.builder()` here, but I think it's essentially the same. I will refactor it a bit to make it more readable.